lib/fetchers/local.rb

```diff
@@ -10,6 +10,7 @@ class Local < Inspec.fetcher(1)
     attr_reader :files
 
     def self.resolve(target)
+      target.gsub!(/\\/, '/')
```

Comment:
instead of mutating the string, could we do a target = target.tr('\\','/') here?